### DISCUSSION TITLE
Theming system

### DISCUSSION THREAD

**Discussion by lucasfth** [PhilipFlyvholm/learnit-plus-plus#31] [Ideas]
If I am not mistaken it seems like the theming system has not been fully implemented yet. 
I would suggest the themes could be injected in the dropdown as seen below:

<img width="1680" alt="image" src="https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/94063597/19657778-af7e-45ab-b516-eade47b62605">

Found in profile -> edit profile -> general info

What do you think?

**Comment by PhilipFlyvholm** [comment]
Would actually be a nice feature to add support in the settings page, I'll add this to the to-do list!
But it should already be possible on the main branch to switch themes using the extension popup, which should show when pressing the extension logo.
<img width="328" alt="image" src="https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/36300927/0ad37e98-b4c9-4b82-8401-7eda7c6b49fc">

In the newest version of Arc they removed the extension bar on the side so now you need to use "cmd+t" and write "learnit++" or press the extension logo in the sidebar in the site control center.
